fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -188,6 +188,24 @@ init_limit_env_vars() {
   fi
 }
 
+init_java_major_version() {
+    # Initialize JAVA_MAJOR_VERSION variable if missing
+    if [ -z "${JAVA_MAJOR_VERSION:-}" ]; then
+        # Parse JAVA_VERSION variable available in containers
+        if [ -n "${JAVA_VERSION:-}" ]; then
+            export JAVA_MAJOR_VERSION="`echo $JAVA_VERSION | sed -E 's/([0-9]+).*/\1/'`"
```

Comment:
`sed -E` is not known by every sed, it's better to use just `-e` which should be good enough for a simple regexp like this.